app/models/user.rb

```diff
@@ -223,7 +223,7 @@ def set_remember_fields
   end
 
   def calculate_score
-    score = (articles.where(featured: true).size * 100) + comments.sum(:score)
+    score = articles.sum(:score) + comments.sum(:score)
```

Comment:
What is the plan for backfilling these to ensure that the scores all get updated and are consistent?